ipaclient/install/client.py

```diff
@@ -101,6 +101,37 @@
 # end of global variables
 
 
+def cleanup(func):
+    def inner(options, tdict):
+        # Add some additional options which contain the temporary files
+        # needed during installation.
+        fd, krb_name = tempfile.mkstemp()
+        os.close(fd)
+        ccache_dir = tempfile.mkdtemp(prefix='krbcc')
+
+        tdict['krb_name'] = krb_name
+        tdict['ccache_dir'] = ccache_dir
+
+        func(options, tdict)
+
+        del os.environ['KRB5_CONFIG']
```

Comment:
I gave you a wrong command :(
KRB5_CONFIG may not be set (for instance if client install is called as part of the server installation), and the code should be instead:
```
os.environ.pop('KRB5_CONFIG', None)
```
to avoid the KeyError exception.

Reply:
Well, I'm the one that didn't test before pushing so...fixed now anyway.